db/migrate/20211204093104_create_batches.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBatches < ActiveRecord::Migration[6.1]
+  def change
+    create_table :batches do |t|
+      t.references :branch
+      t.references :coaching
+      t.string :coach_name
+      t.integer :players_limit
+      t.integer :players_count
+      t.integer :start_at_in_HH
+      t.integer :end_at_in_HH
```

Comment:
dont store just the hour. your code should be future-proof. what if in future, I want to have a batch starting at 07:32:35 to 08:32:35. there should be a way to store this

Reply:
Now start_at and end_at stored in time data ttype.